api/gpt.py

```diff
@@ -33,11 +38,23 @@ class GPT:
 
     def __init__(self, engine='davinci',
                  temperature=0.5,
-                 max_tokens=100):
+                 max_tokens=100,
+                 input_prefix = "input: ",
+                 input_suffix = "\n",
+                 output_prefix = "output: ",
+                 output_suffix = "\n",
+                 append_input_suffix_and_output_prefix_to_query = False,
+                 stop = "\ninput:"):
         self.examples = []
         self.engine = engine
         self.temperature = temperature
         self.max_tokens = max_tokens
+        self.input_prefix = input_prefix
+        self.input_suffix = input_suffix
+        self.output_prefix = output_prefix
+        self.output_suffix = output_suffix
+        self.append_input_suffix_and_output_prefix_to_query = append_input_suffix_and_output_prefix_to_query
```

Comment:
Can you document or explain what this parameter, `append_input_suffix_and_output_prefix_to_query`,  could be used for? I understand why you may want the `output_prefix` to be optional in the priming text, but to format the priming properly, I think you may always need the `input_suffix` in the priming. But I could be wrong -- would love to hear your thoughts here!

Reply:
Totally agree - I think I'd got in a muddle with this trying to replicate existing offsets ( see comments below about the change to the value now being sent for stop,  and the change to the offset).    I've now changed parameter name to append_output_prefix_to_query,  which now your suggested changes have been made is used to do exactly that - to add the output prefix only.

Reply:
Sounds good!